src/components/auth/login/SignUpScreen.js

```diff
@@ -43,13 +46,15 @@ const SignupScreen = ({ isSignup, screenProps, styles, handleLoginMethod, sdkIni
   const _mobile = () => {
     const { current: captcha } = reCaptchaRef
 
+    !isMobileNative && setRecaptchaVisible(true)
+
     // If recaptcha has already been passed successfully, trigger torus right away
-    if (captcha.hasPassedCheck()) {
+    if (captcha?.hasPassedCheck()) {
```

Comment:
not sure do we have webpack polyfills for that.
just add 

```
if (!captch) {
  return
}
```

just after the const assignment